tests/conftest.py

```diff
@@ -116,3 +119,33 @@ def virtual_drive():
     virtual_servo = net.connect_to_slave(server.ip, server.dictionary_path, server.port)
     yield server, virtual_servo
     server.stop()
+
+
+@pytest.fixture(scope="session")
+def connect_to_rack_service():
+    rack_service_port = 33810
+    client = rpyc.connect("localhost", rack_service_port, config={"sync_request_timeout": None})
+    yield client.root
+    client.close()
+
+
+@pytest.fixture(scope="session", autouse=True)
+def load_firmware(pytestconfig, read_config, request):
```

Comment:
are you sure we want autouse? Aren't there unit tests that run without any drive...?

Reply:
Yes, I am.  For the no_connection tests, the fixture does nothing.